src/nft/css/sprinkles.css.ts

```diff
@@ -22,6 +22,7 @@ const themeContractValues = {
     modalBackdrop: '',
 
     stateOverlayHover: '',
+    stateOverlayPressed: '',
```

Comment:
This and stateOverlayHover are the same in dark and light mode and can be removed from the themeVars and respective theme files

Reply:
It also looks like `stateOverlayPressed` hasn't been used anywhere

Reply:
There is a task to update the hover states.  I started working on it and then realized it would look bad. So I requested a figma design.  I will delete this for now